server/api/e2e/gql_document_test.go

```
@@ -0,0 +1,466 @@
+	case int:
+		version = v
+	case float64:
+		version = int(v)
+	default:
+		return "", 0, false
+	}
+
+	return projectID, version, true
+}
+
+func documentSeeder(ctx context.Context, r *repo.Container) error {
```

golangci-lint (1.62.2)

func `documentSeeder` is unused

(unused)